gwswlib/threedi.py

```diff
@@ -236,10 +333,9 @@ def get_code(self, connection, default_code=""):
         ]
         connection_number = len(nr_connections) + 1
```

Comment:
```suggestion
        connection_count = len(nr_connections) + 1
```

"number" klinkt als "nummer drie uit een lijst van vier", vandaar.
Maar misschien is het wel precies wat je bedoelt, omdat je `len(...) + 1` doet.